sale_quotation_number/models/sale_order.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# © 2010-2012 Andy Lu <andy.lu@elico-corp.com> (Elico Corp)
+# © 2013 Agile Business Group sagl (<http://www.agilebg.com>)
+# © 2017 valentin vinagre  <valentin.vinagre@qubiq.es> (QubiQ)
+# © 2020 Manuel Regidor  <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import models, api
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    @api.model
+    def create(self, vals):
+        company = self.env['res.company']._company_default_get('sale.order')
```

Comment:
company might be in vals - if so, must use that in preference...

Reply:
Done.